index.html

```diff
@@ -51,6 +51,109 @@ <h1 class="introduction">I'm Ivan<br>Glad to see you</h1>
+                    <strong class="year">2015</strong>
+                </div>
+                <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <img src="./images/html.png" alt="html tag" class="work-tags">
+                    <img src="./images/css.png" alt="css tag" class="work-tags">
+                    <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+                </div>
+                <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+            </div>
+        <div class="work">
+            <img src="./images/work3.svg" alt="screenshot of first project" class="work1_pic">
+            <h3 class="work-name">Tonic</h3>
+            <div class="work-data">
+                <strong class="company"> Canopy </strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="role">Back-End Dev</strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="year">2015</strong>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <img src="./images/html.png" alt="html tag" class="work-tags">
+                <img src="./images/css.png" alt="css tag" class="work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+            </div>
+            <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+        </div>
+        <div class="work">
+            <img src="./images/work4.svg" alt="screenshot of first project" class="work1_pic">
+            <h3 class="work-name">Multi-Post Stories</h3>
+            <div class="work-data">
+                <strong class="company"> Canopy </strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="role">Back-End Dev</strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="year">2015</strong>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <img src="./images/html.png" alt="html tag" class="work-tags">
+                <img src="./images/css.png" alt="css tag" class="work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+            </div>
+            <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+        </div>
+    </section>
+    <section class="about">
```

Comment:
- [ ] Please make sure that your `about` section it's aligned with flex as this is part of the requirements.

Reply:
Sorry, I did not pay attention to this one. Corrected it now.